wp-content/themes/core/components/blocks/interstitial/Interstitial_Block_Controller.php

```diff
@@ -0,0 +1,202 @@
+<?php
+declare( strict_types=1 );
+
+namespace Tribe\Project\Templates\Components\blocks\interstitial;
+
+use Tribe\Libs\Utils\Markup_Utils;
+use Tribe\Project\Blocks\Types\Hero\Model;
+use Tribe\Project\Blocks\Types\Interstitial\Controller;
+use Tribe\Project\Blocks\Types\Interstitial\Interstitial as Interstitial_Block;
+use Tribe\Project\Templates\Components\Abstract_Controller;
+use Tribe\Project\Templates\Models\Image;
+use Tribe\Project\Theme\Config\Image_Sizes;
+
+/**
+ * Class Interstitial
+ */
+class Interstitial_Block_Controller extends Abstract_Controller {
+
+	public const LAYOUT            = 'layout';
+	public const MEDIA             = 'media';
+	public const CONTENT           = 'content';
+	public const CTA               = 'cta';
+	public const CONTAINER_CLASSES = 'container_classes';
+	public const MEDIA_CLASSES     = 'media_classes';
+	public const CONTENT_CLASSES   = 'content_classes';
+	public const CLASSES           = 'classes';
+	public const ATTRS             = 'attrs';
+
+	public string $layout;
+	public ?int $media;
+	public string $content;
+	public array $cta;
+	public array $container_classes;
+	public array $media_classes;
+	public array $content_classes;
+	public array $classes;
+	public array $attrs;
+
+	public function __construct( array $args = [] ) {
+		$args                    = $this->parse_args( $args );
+		$this->classes           = (array) $args[ self::CLASSES ];
+		$this->layout            = $args[ self::LAYOUT ];
+		$this->media             = $args[ self::MEDIA ];
+		$this->content           = $args[ self::CONTENT ];
+		$this->cta               = $args[ self::CTA ];
+		$this->container_classes = (array) $args[ self::CONTAINER_CLASSES ];
+		$this->media_classes     = (array) $args[ self::MEDIA_CLASSES ];
+		$this->content_classes   = (array) $args[ self::CONTENT_CLASSES ];
+		$this->attrs             = (array) $args[ self::ATTRS ];
+		$this->init();
+	}
+
+	/**
+	 * @return array
+	 */
+	protected function defaults(): array {
+		return [
+			self::LAYOUT            => Interstitial_Block::LAYOUT_LEFT,
+			self::MEDIA             => null,
+			self::CONTENT           => '',
+			self::CTA               => [],
+			self::CONTAINER_CLASSES => [ 'b-interstitial__container', 'l-container' ],
```

Comment:
Hmm... some of these should be required classes. (I think this affected Hero & Interstitial too.)